cli.js

```diff
@@ -8,12 +8,115 @@ const { program } = require('commander')
+const TurndownService = require('turndown')
+const {tables}  = require('turndown-plugin-gfm')
+
+// Languages that can be specified in the code markdown
+const codeLanguages = {
+  css: 'css',
+  bash: 'bash',
+  php: 'php',
+  yaml: 'yaml',
+  xml: 'xml',
+  jscript: 'javascript',
+}
+
+// Rules that remove escapes in code blocks
+const unEscapes = [
+  [/\\\\/g, '\\'],
+  [/\\\*/g, '*'],
+  [/\\-/g, '-'],
+  [/^\\+ /g, '+ '],
+  [/\\=/g, '='],
+  [/\\`/g, '`'],
+  [/\\~~~/g, '~~~'],
+  [/\\\[/g, '['],
+  [/\\\]/g, ']'],
+  [/\\>/g, '>'],
+  [/\\_/g, '_'],
+  [/\&quot;/g, '"'],
+  [/\&lt;/g, '<'],
+  [/\&gt;/g, '>'],
+]
+
+const turndownService = new TurndownService({
   headingStyle: 'atx',
   codeBlockStyle: 'fenced',
   emDelimiter: '*',
 })
+turndownService.use(tables);
+
+// Remove Glossary
+turndownService.addRule('glossary', {
+  filter: node => {
+    const classList = node.getAttribute('class');
+    if ( classList ) {
+      return classList === 'glossary-item-hidden-content';
+    }
+    return false;
+  },
+  replacement: () => {
+    return '';
+  }
+});
+
+// Remove code trigger anchor
+turndownService.addRule('code-trigger-anchor', {
+  filter: node => {
+    const classList = node.getAttribute('class');
+    if ( classList ) {
+      return classList.includes('show-complete-source') || classList.includes(`less-complete-source`);
+    }
+    return false;
+  },
+  replacement: () => {
+    return '';
+  }
+});
+
+// Transform dt tag to strong tag
+turndownService.addRule('dt-to-strong', {
+  filter: ['dt'],
+  replacement: (content, node, options) => {
+    return options.strongDelimiter + content + options.strongDelimiter
+  }
+});
+
+// Transform pre code block to code markdown
+turndownService.addRule('precode to code', {
+  filter: node => {
+    const classList = node.getAttribute('class');
+    const isCode = node.nodeName === 'PRE' && classList && classList.includes('brush:');
+    return isCode;
+  },
+  replacement: (content, node) => {
+    const classList = node.getAttribute('class');
+
+    // Search for a language that is match the list of code languages
+    const codeLanguage = Object.keys(codeLanguages).reduce((currentLanguage, language) => {
+      if ( classList.includes(language) ) {
+        return codeLanguages[language];
+      }
+      return currentLanguage;
+    }, undefined);
+
+    // Unescape contents
+    let newContent = unEscapes.reduce((accumulator, unEscape) => {
+      return accumulator.replace(unEscape[0], unEscape[1])
+    }, content)
+
+    // Remove br tag
+    newContent = newContent.replace(/^<br \/>\n\n|<br \/>\n/g, "\n");
+    // Remove first and last paragraph tag
+    newContent = newContent.replace(/^<\/p>|<p>$/g, '');
+    // Remove first new line
+    newContent = newContent.replace(/^\n/, '');
+    // Convert to language-aware markdown
+    newContent = codeLanguage ? `\`\`\`${codeLanguage}\n` + newContent + '```' : "```\n" + newContent + '```';
```

Comment:
```suggestion
    newContent = '```' + (codeLanguage ?? '') + "\n" + newContent + '```';
```
こちらの方がコードの繰り返しが少なく読みやすいと思いました